Cargo.lock

```diff
@@ -1485,6 +1513,12 @@ dependencies = [
  "termcolor",
 ]
 
+[[package]]
+name = "gcc"
+version = "0.3.55"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "8f5f3913fa0bfe7ee1fd8248b6b9f42a5af4b9d65ec2dd2c3c26132b950ecfc2"
```

Comment:
The gcc crate has been renamed to cc 4 years ago. I believe the latest version of the gcc crate doesn't support eg AArch64 macOS. Maybe use the twox-hash crate instead? It is a pure rust implementation, so it doesn't need a C compiler. (Don't use the xxhash-rust crate at it is licensed as BSL-1.0 which is way too restrictive for rust)

Reply:
Indeed. Thanks for pointing it out! I've changed it to use the twox-hash crate instead.